src/linter/custom.go

```diff
@@ -176,14 +179,14 @@ var vscodeLevelMap = map[int]int{
 	// LevelSyntax is intentionally not included here
 }
 
-var severityNames = map[int]string{
-	LevelError:       "ERROR  ",
-	LevelWarning:     "WARNING",
-	LevelInformation: "INFO   ",
-	LevelHint:        "HINT   ",
-	LevelUnused:      "UNUSED ",
-	LevelDoNotReject: "MAYBE  ",
-	LevelSyntax:      "SYNTAX ",
+var severityLevelNames = map[SeverityLevel]string{
+	LevelError:       "error",
+	LevelWarning:     "warning",
+	LevelInformation: "info",
+	LevelHint:        "hint",
+	LevelUnused:      "unused",
+	LevelDoNotReject: "maybe",
+	LevelSyntax:      "syntax",
```

Comment:
I'm not sure about this severity level distribution. If you use min level filtering, you need a good sorting here and I don't know how to rank them from 1 to 7 properly. Are you sure you don't want to filter by using string list of prefixes?

Reply:
I'm agree, it's better to pass enabled severity list.
However, it's inconvient to pass levels with command line flags.
So I suggest:
1) add config file support with separate PR
2) after that, add filtering by severity level names